cli/src/main/kotlin/tools/samt/cli/CliCompiler.kt

```diff
@@ -3,11 +3,12 @@ package tools.samt.cli
 import tools.samt.common.DiagnosticConsole
 import tools.samt.common.DiagnosticContext
 import tools.samt.lexer.Lexer
+import tools.samt.parser.FileNode
 import tools.samt.parser.Parser
 import tools.samt.parser.ParserException
 import java.io.File
 
-fun parse(files: List<String>) {
+fun parse(files: List<String>, dumpAST: Boolean = false) {
```

Comment:
Given seperation of duties, it would be a lot cleaner to have the `dumpAST` only within the CLI module, we don't really want a side-effect like writing to the standard out within the Parser, this would be dubious given the language server integration in the future